packages/graphql/src/schema/query.ts

```diff
@@ -261,7 +284,8 @@ export const Query = builder.queryType({
           return SessionService.getPinnedFeedbacks(args, ctx)
         },
       }),
-      course: asAuthenticated.field({
+
+      course: asUser.field({
```

Comment:
Changed `asAuthenticated` to `asUser` as this query is only used in `frontend-manage`